.github/workflows/ci.yml

```diff
@@ -157,6 +157,7 @@ jobs:
     strategy:
       matrix:
         scenario:
+          - 'ember-lts-3.20'
```

Comment:
Can we list `ember-lts-3.20` after `ember-lts-3.16`? (personal preference)

Reply:
Done!